addons/binding/create_openhab_binding_skeleton.cmd

```diff
@@ -11,7 +10,18 @@ IF %ARGC% NEQ 2 (
 	exit /B 1
 )
 
-mvn archetype:generate -N -DarchetypeGroupId=org.eclipse.smarthome.archetype -DarchetypeArtifactId=org.eclipse.smarthome.archetype.binding -DarchetypeVersion=0.9.0-SNAPSHOT -DgroupId=org.openhab.binding -DartifactId=org.openhab.binding.%2 -Dpackage=org.openhab.binding.%2 -Dversion=2.1.0-SNAPSHOT -DbindingId=%2 -DbindingIdCamelCase=%1 -DvendorName=openHAB -Dnamespace=org.openhab
+mvn -s ../archetype-settings.xml archetype:generate -N \

```

Comment:
@jwveldhuis Thanks for testing. I revert adding the linebreaks. Sry for my headlessness. Didn't thought about that.